src/FlipMove.js

```diff
@@ -69,7 +69,7 @@ class FlipMove extends Component {
     // can complete. Because we cannot mutate props, we make `state` the source
     // of truth.
     this.state = {
-      children: props.children.map(child => ({
+      children: Children.toArray(props.children).map(child => ({
```

Comment:
In `flip-move.js`, we have the following:

```jsx
render() {
  return (
    <ComposedComponent {...this.convertProps(this.props)} />
  );
}
```

But that is really the same as doing this:

```jsx
render() {
  return React.createElement(ComposedComponent, this.convertProps(this.props));
}
```

So the props are passed back to React, which handles calling the constructor and lifecycle callbacks of `ComposedComponent`.  React may not pass the same props object to `ComposedComponent` that was passed to `React.createElement`, in particular, it is free to change the children object from an array to whatever.  That is what preact does, hence the issue.

That means we need to do the conversion of children to an array in `FlipMove`.